frontend/src/components/room/RoomPage.js

```diff
@@ -125,14 +140,17 @@ function RoomPage({ socket }) {
               <Grid item xs={2} display="flex" justifyContent="flex-end">
                 <Paper varient={6}>
                   <Typography variant={"h5"} m={"5px"}>
-                    {difficultyLevel ?? "unknown diff"}
+                    {question.difficulty ?? "unknown diff"}
                   </Typography>
                 </Paper>
               </Grid>
             </Grid>
           </Box>
           <Paper variant="outlined" square>
-            <Typography sx={{ height: "40rem" }}>Question</Typography>
+            <Typography sx={{ height: "1.5rem", textDecoration: "underline" }}>
+              {question.title}
+            </Typography>
```

Comment:
<img width="659" alt="Screenshot 2022-10-09 at 12 05 28 AM" src="https://user-images.githubusercontent.com/66197002/194716736-6f64c598-d04b-47df-bd1e-f38bee74fc6e.png">

I have update a new design. the title should be at the top of the box as shown in the picture. I will update it in my pr maybe you could remove this 3 lines from your pr. Thank you

Reply:
https://github.com/CS3219-AY2223S1/cs3219-project-ay2223s1-g23/pull/34/commits/63f0b4ca306d862d3a6d2c80a511896d64a5e06e